week-9/Homework/mandatory/1-practice/2-code-reading.md

```diff
@@ -47,7 +51,7 @@ function f1(val) {
 }
 
 f1(x);
-console.log(x);
+console.log(x); (I am not sure why it outputs 9)
```

Comment:
It outputs 9 because nothing happens to x after it is declared on line 46, the `f1()` function is called but that only adds 1 to variable `val` which is declared inside the function.

`f1(x)` would return 10 but that is not what is being logged to the console :)